clients/ui/frontend/src/app/pages/modelCatalogSettings/screens/CatalogSourceConfigsTable.tsx

```diff
@@ -14,32 +24,83 @@ type CatalogSourceConfigsTableProps = {
 const CatalogSourceConfigsTable: React.FC<CatalogSourceConfigsTableProps> = ({
   catalogSourceConfigs,
   onAddSource,
-}) => (
-  <Table
-    data-testid="catalog-source-configs-table"
-    data={catalogSourceConfigs}
-    columns={catalogSourceConfigsColumns}
-    toolbarContent={
-      <Toolbar>
-        <ToolbarContent>
-          <ToolbarItem>
-            <Button
-              variant="primary"
-              icon={<PlusCircleIcon />}
-              onClick={onAddSource}
-              data-testid="add-source-button"
-            >
-              Add a source
-            </Button>
-          </ToolbarItem>
-        </ToolbarContent>
-      </Toolbar>
+}) => {
+  const [toggleError, setToggleError] = React.useState<Error | undefined>(undefined);
+  const [isUpdatingToggle, setIsUpdatingToggle] = React.useState(false);
+  const { apiState, refreshCatalogSourceConfigs } = React.useContext(ModelCatalogSettingsContext);
+  const [showAlert, setShowAlert] = React.useState<boolean>(false);
```

Comment:
The `showAlert` state is redundant. I think we can just rely on the `toggleError` state for whether to show the alert, and when we want to hide the alert just `setToggleError(undefined)`.